### DISCUSSION TITLE
Need Authenticator file or not?

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#43792] [Q&A] [answered]
Do we not need `Authenticator` file to create the login form and authentication? Because nothing is said about `Authenticator` file here https://symfony.com/doc/current/security.html#form-login
for ex `src/Security/SecurityAuthenticator.php`

**Comment by stof** [comment, marked as answer]
If you use the `form_login` configuration, you don't need to create an Authenticator. The authenticator for `form_login` is implemented in Symfony ([here](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Security/Http/Authenticator/FormLoginAuthenticator.php))

**Reply by karaka200** [reply]
Is it secure that use `form_login` ?
